services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from django.utils.dateparse import parse_date
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: int |float,
+                         movie_id: id,
+                         cinema_hall_id: int) -> MovieSession:
```

Comment:
The type hint for `movie_show_time` is incorrect. It should be a `datetime`, not `int | float`, as it represents a date and time.